src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/InputStep/help-submitter.html

```diff
@@ -1,4 +1,5 @@
 <div>
     User IDs and/or <em>external</em> group names of person or people permitted to respond to the input, separated by ','.
-    Spaces will be trimmed. This means that "alice, bob, blah " will be equivalent to "alice,bob,blah".
+    Spaces will be trimmed. This means that "alice, bob, blah " will be equivalent to "alice,bob,blah".<br>
+    <strong>If you are an administrator the restriction does not apply and you are permitted to respond regardless if your are in the list or not.</strong>
```

Comment:
Thanks for filing a PR! I might reword this a bit, but I'd be interested to hear what @bitwiseman thinks would work well here. Here's how I would phrase it:

```suggestion
    <strong>Note: Jenkins administrators are able to respond to the input regardless of the value of this parameter.</strong>
```

Reply:
Yup, what @dwnusbaum said.